### ISSUE TITLE
Shouldn't standard package be a dev dependecie?

### ISSUE THREAD

**Issue by cedced19** [Kikobeats/exists-file#3]
I saw it [there](https://github.com/Kikobeats/exists-file/blob/master/package.json#L27).
Standard is a validation test CLI so that's why I don't think that there is no need to be included in dependecies.
Am I right?

**Comment by Kikobeats** [comment]
totally right, It was my faul. I'm going to release a new version now!

**Comment by cedced19** [comment]
Ok thank you ! It is a good point for me to have less dependencies when I do application with electron.

**Comment by Kikobeats** [comment]
@cedced19 I was reviewing the API of the lib and I'm thinking in release a v2 with this methods:
- existsFile
- existsFileSync

This are more align with nodejs API. What do you think?

**Comment by cedced19** [comment]
Yes it's more aligned with nodejs APIs.
But how do you think it will work to require functions?
Something like that ?

``` javascript
var existsFile = require('exists-files');
var existsFileSync = existsFile.sync;
```

It can be harder to use, do you think so ?

**Comment by jviotti** [comment]
Alternatively, you can try to be a bit smart and detect whether a callback was passed as the last argument to the function.

The `.sync` convention is well understood though, I'd go with that.

**Comment by cedced19** [comment]
I think you can let as it is or you can do what @jviotti said.

**Comment by Kikobeats** [comment]
Currently the API is based in the cb detection to determinate if use `sync` or `async` method.

The problem of this approach is that not possible use `async` without waiting a callback.

Sure that you can provide a `noop` callback but in other case `sync` method will be executed.

I said about use `existsFileSync` and `existsFile` because internally the library is based in `fs.stat` and `fs.statSync`, have sense 😄 .

**Comment by cedced19** [comment]
Why not use two functions but I'm asking myself how you can provide two functions with `module.exports` that's why we thought about `.sync`.

**Comment by Kikobeats** [comment]
@cedced19 

functions works as object, so you can do:

```
function noop() {}
noop.sync = function() {}
module.exports = noop
```

**Comment by cedced19** [comment]
That's what I mean. Now I'm sure we are talking about the same thing. Use different functions are great because the part were you detect if there is a callback is removed. It will be more powerful.

**Comment by Kikobeats** [comment]
completely, also faster because it removes if statement.

Released under `v2.0.0`, thanks guys! 💪
